src/components/Books.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import store from '../redux/configureStore';
+import BookCard from './BookCard';
+
+const Books = () => {
+  const {
+    books: { books },
+  } = store.getState();
+  const Books = books.map((book) => {
+    const {
+      title,
+      author,
+      category,
+      completed,
+    } = book;
+    return (
+      <BookCard
+        key={Math.random()}
```

Comment:
- [ ] I don't think Math.random() is a good way to generate key. Your app is throwing a console error related to children with same key. You can use uuid instead. Kindly fix this issue. :pray: 

![Screenshot from 2021-11-03 01-54-09](https://user-images.githubusercontent.com/64894927/139963130-7c7897a1-0725-49fc-a9fe-e60c117eb2f1.png)